traffic_mirroring.go

```diff
@@ -30,36 +25,36 @@ func main() {
 	}
 
 	packetSource := gopacket.NewPacketSource(handle, handle.LinkType())
-	max_size := 10
-	packets_list := make([][]byte,max_size) 
-	current_packet := 0
+	packetsList := make(chan []byte, 22)
+	filePath := "Test"
+	go writeFile(packetsList, filePath)
 
 	for overlayPacket := range packetSource.Packets() {
```

Comment:
Could you extract this `for` implementation to a function? Doing that, you are going to be able to unit test your code, and catch bugs earlier.